### PR TITLE
Add integration testing

### LINKED ISSUE #27: Test that Form elements are rendered as expected

### PR DESCRIPTION
**Pull request by Atos20**
#### What's this PR do?  
- It provides `unit testing`  to the `<ActivityCards>` component
- It provides `unit testing` and `integration testing` to `<DetailsPage>`
- it provides `unit testing` for `<ResultPage>` component

#### Where should the reviewer start? 
- `ActivityCards.js`
- `DetailsPage.js`
- `ResultPages.js`

#### How should this be manually tested?  
 On the terminal run `npm test`. All tests should fail except for the file mentioned above.

#### Any background context you want to provide?  

We are aiming to have a fully tested application and this PR complements the work of my other two colleagues.  Unit testing aims to make sure the components and elements render accordingly.

#### What are the relevant tickets?  
closes #27 
#### Screenshots (if appropriate) 
Screen Shot 2020-11-04 at 7 26 16 PM
Screen Shot 2020-11-04 at 7 26 40 PM
Screen Shot 2020-11-04 at 7 27 08 PM

### CODE REVIEW

**Review by BlakeDonn — APPROVED** [review]
These tests look to cover a good portion of our user interaction as it relates to the respective views. Good job!

File: src/Components/ResultPage/ResultPage.test.js [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+        expect(determineRandomActivity).toHaveBeenCalled();
+
+        const movieTitle2  =  await waitFor(() => screen.getByText('Corpse Bride'))
+        const moviePublic2  =  await waitFor(() => screen.getByText('PG'))
+        const randomImg2 = await waitFor(() => screen.getByRole('img'));
+        const simmilarPicksTitle2 = screen.getByRole('heading', { name: /simmilar picks/i });
+        const choosemeButton2 = screen.getByRole('button', { name: /choose me!/i });
+        const skipButton2 = screen.getByRole('button', { name: /skip/i });
+
+        expect(movieTitle2).toBeInTheDocument();
```
Comment (nicolegooden) [inline src/Components/ResultPage/ResultPage.test.js:216]: Amazing testing here, very thorough and thanks for covering the edge cases we've implemented.

**Review by nicolegooden — APPROVED** [review]
